### PR TITLE
I made updates

### PR DESCRIPTION
**Pull request by mstonerock**
tl:dr pull

### CODE REVIEW

**Comment by mstonerock** [comment]
asdfasdfsad

File: _slides/04-mstonerock.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (SCHJihong) [inline _slides/04-mstonerock.md:3]: Great title!

File: _slides/04-mstonerock.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 ---
 
-Caption here
+BOOYAH!!!!
 
-![droidtocat](https://octodex.github.com/images/droidtocat.png)
+![droidtocat](https://octodex.github.com/images/droidtocat.png)
```
Comment (SCHJihong) [inline _slides/04-mstonerock.md:8]:
```suggestion
![droidtocat](https://octodex.github.com/images/droidtocat.png)
```
Test suggestion.

**Review by SCHJihong — APPROVED** [review]
Approved!

File: _slides/04-mstonerock.md (hunk 1 above)
Comment (sch-pv) [reply _slides/04-mstonerock.md:3]: bad comments they should be meaningful.

**Comment by sch-pv** [comment]
test

**Pull request closed without merging**